collectors/ebpf.plugin/README.md

```diff
@@ -116,8 +188,33 @@ processes](https://en.wikipedia.org/wiki/Zombie_process).
 
 #### Task error
 
-The functions responsible for ending tasks do not return values, so this chart contains information about failures on
-process and thread creation.
+The functions responsible for ending tasks do not return values, so this chart contains information about failures on 
+process and thread creation only.
+
+#### TCP functions
+
+This chart demonstrates calls to functions `tcp_sendmsg`, `tcp_cleanup_rbuf` and `tcp_close`, these functions are
+used to send, and receive data and to close connections when `TCP` protocol is used.
+
+#### TCP bandwidth
+
+Like the previous section, this chart also monitors `tcp_sendmsg` and `tcp_cleanup_rbuf` but, instead of showing number of calls,
+it demonstrates the number of bytes sent and received.
+
+#### TCP retransmit
+
+This chart demonstrates calls to functions `tcp_retransmit` that is responsible to execute the TCP retransmission when the
+receiver did not return the package during the expected time.
+
+#### UDP functions
+
+This chart demonstrates calls to  functions `udp_sendmsg`, and `udp_recvmsg` responsible to send and receive data
+for connections when `UDP` protocol is used.
+
+#### UDP bandwidth
+
+Like previous section, this chart also monitors `tcp_sendmsg` and `tcp_cleanup_rbuf`but,  instead of showing the number of calls,
```

Comment:
```diff
-Like previous section,
+Like the previous section,
```

```diff
-`tcp_sendmsg` and `tcp_cleanup_rbuf`but,  instead of
+`tcp_sendmsg` and `tcp_cleanup_rbuf`, but instead of
```